examples/decode.py

```diff
@@ -14,18 +14,25 @@ def main():
                         help='Output file (config.xml)')
     parser.add_argument('--key', type=lambda x: x.encode(), default=b'',
                         help="Key for AES decryption")
+    parser.add_argument('--serial', type=str, default='',
```

Comment:
I think this would be better as an alias for --key, and then you can add a new argument for --digi with action='store_true'?

Reply:
I though about that but I prefer a different parameter because:

- it's not actually the key, it **is** the serial number of the router used to derive the keys

- it's a string and not bytes